### PR TITLE
Adding `fractal-client`

### PR DESCRIPTION
**Pull request by tcompa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tcompa** [comment]
@conda-forge/help-python, ready for review!

File: recipes/fractal-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - python-dotenv >=0.20.0,<0.21.0
+    - rich >=12.5.1,<13.0.0
+    - httpx >=0.23.0,<0.24.0
+    - pyjwt >=2.4.0,<3.0.0
+    - packaging >=23.1.0,<24.0.0
+
+test:
+  imports:
+    - fractal_client
+  commands:
```
Comment (synapticarbors) [inline recipes/fractal-client/meta.yaml:35]: Can you put in a check of the cli under the `commands` test. Something like `- fractal --help` or similar.

**Review by synapticarbors — COMMENTED** [review]
One minor request, but otherwise looks good.

File: recipes/fractal-client/meta.yaml (hunk 1 above)
Comment (tcompa) [reply recipes/fractal-client/meta.yaml:35]: Of course, thanks.
Fixed with https://github.com/conda-forge/staged-recipes/pull/24101/commits/fb09f00875a0d5a63d36fdc7ecbce9bd8eded479.

**Comment by tcompa** [comment]
I implemented the request change.

@conda-forge/help-python PR ready for review. Thanks!

**Pull request merged**